src/cli/format.js

```diff
@@ -227,8 +227,13 @@ function format(context, input, opt) {
 }
 
 async function createIgnorerFromContextOrDie(context) {
+  const filepath = context.argv.filepath
```

Comment:
Where this filepath come from? There is no `filepath` flag.

Reply:
There is - as [`--stdin-filepath`](https://github.com/prettier/prettier/blob/fa3b95c2e1a3a3351dc1b0168aa2c26f0cc4e9b3/src/main/core-options.js#L97-L106) - and it's used [here](https://github.com/prettier/prettier/blob/fdccc0f48e5bd3a17ae1e3500c1b5c1763c6a368/src/cli/index.js#L99) as well.

Reply:
`filepath` only exists when format stdin, when formatting files, it doesn't exist.

Reply:
See `formatFiles` function, it calls `createIgnorerFromContextOrDie` on first line. I don't think it will work.